cases/function/function/test_udaf_function.yaml

```diff
@@ -136,27 +138,27 @@ cases:
     expect:
       order: id
       columns: ["id int","c1 string","m2 double","m3 double","m4 double","m5 double","m6 double", "m7 double"]
-      data: |
-        1, aa, 1,   NULL, 30,   1.100000023841858,2.1,               NULL
-        2, aa, 2.5, 4.0,  31.5, 1.25,             2.25,              5.0
-        3, aa, 2,   2.5,  32,   1.200000007947286,2.1999999999999997,3.5
-        4, aa, 2.5, 2.5,  33,   1.25,             2.25,              3.5
+      rows:
+        - [1, aa, 1,   NULL, 30,   1.100000023841858,2.1,               NULL]
+        - [2, aa, 2.5, 4.0,  31.5, 1.25,             2.25,              5.0]
+        - [3, aa, 2,   2.5,  32,   1.200000007947286,2.1999999999999997,3.5]
+        - [4, aa, 2.5, 2.5,  33,   1.25,             2.25,              3.5]
   -
     id: 5
     desc: distinct_count
     sqlDialect: ["HybridSQL"]
     inputs:
       -
-        columns : ["id int","c1 string","c2 smallint","c3 int","c4 bigint","c5 float","c6 double","c7 timestamp","c8 date","c9 string","c10 bool"]
-        indexs: ["index1:c1:c7"]
+        columns : ["id int","c1 string","c2 smallint","c3 int","c4 bigint","c5 float","c6 double","c7 timestamp","c8 date","c9 string","c10 bool","ts timestamp"]
+        indexs: ["index1:c1:ts"]
```

Comment:
?

Reply:
A TS column is added separately. The time of the TS column is not repeated, and the C7 column has a repeated timestamp to test the function